content/r-package-dev/Documentation.Rmd

```diff
@@ -153,77 +153,119 @@ is.valid.pH <- function(pH){
-#' is.valid.pH(5)
-#' is.valid.pH(-5)
-#' is.valid.pH(1:15)
-#' 
-#' set.seed(5)
-#' pH_vals <- runif(5, -3, 20)
-#' is.valid.pH(pH_vals)
-#' 
-#' \dontrun{
-#' is.valid.pH()
-#' }
-#'
-is.valid.pH <- function(pH){
-    pH >= 0 & pH <= 14
-}
-```
 
 Now that your roxygen comments have been added, you need to build the resulting `.Rd` files. To manually build your help files, execute `roxygen2::roxygenize()`. To setup your *Build Tools* to automatically convert roxygen to `.Rd` each time you build and reload the package: 
 
 1. Go to the *Build Tab*, click *More*, and then *Configure Build Tools*.
-2. Click the checkmark next to *Generate documentation with Roxygen*. 
+2. Click the check mark next to *Generate documentation with Roxygen*. 
 3. Next to *Generate documentation with Roxygen*, click *Configure...* and make sure that everything is checked. Caveat: if you create long-running [vignettes](#vignettes), you might want to uncheck that field so that they don't knit every time you build the package.
 
 Once you build your `.Rd` files, you will see them appear in the `man` folder. Each function will have its own `.Rd`. You should now be able to use `?functionName` and see your help file. For the example we have been following, we should be able to see our help file by executing `?is.valid.pH`. You will also see a new file, `NAMESPACE` in your top-level directory. This is automatically generated from the roxygen comments - don't edit it.
 
-<name="pkg-level"</a>
+<a name="pkg-level"</a>
 
 ### Package-level help file
 
-These are optional help files that don't match a specific function. They can be about the whole package, or document datasets available in the package. The developer can name them anything using that roxygen tag `@name`, and users access them with `?name`. They are written with similar roxygen syntax, but instead of the function you use `NULL`. Here's an example:
+It is optional, but nice to the user to have a "package-level" help file. Users are use to looking for help by typing a `?` and function name, but if they type `?` and package name, it's nice to give them information on the package. Here is a modified example of the `dataRetrieval` package-level help file:
 
 ```{r eval=FALSE}
-#' @title My top-level help file
-#' @description Some additional information that should be available to users.
-#' @name toplevel
-#' @author My Name
+#' Retrieval functions for USGS and EPA data
+#'
+#' \tabular{ll}{
+#' Package: \tab dataRetrieval\cr
+#' Type: \tab Package\cr
+#' License: \tab Unlimited for this package, dependencies have more restrictive licensing.\cr
+#' Copyright: \tab This software is in the public domain because it contains materials
+#' that originally came from the United States Geological Survey, an agency of
+#' the United States Department of Interior. For more information, see the
+#' official USGS copyright policy at
+#' https://www.usgs.gov/visual-id/credit_usgs.html#copyright\cr
+#' LazyLoad: \tab yes\cr
```

Comment:
LazyLoad? We do talk about it in Package Mechanics. Or are you talking about something else?